content/docs/gke/authentication.md

```diff
@@ -0,0 +1,181 @@
++++
+title = "Authenticating Kubeflow to GCP"
+description = "Authentication and authorization to GCP"
+weight = 4
++++
+
+
+# Authenticating gcloud
+
+[The `gcloud` tool](https://cloud.google.com/sdk/gcloud/) is used to interact with Google Cloud Platform (GCP) over the command line. 
+You can use the `gcloud` command to [set up Google Kubernetes Engine (GKE) clusters](https://cloud.google.com/sdk/gcloud/reference/container/clusters/create), 
+and interact with other Google services.
+
+### Logging in
+
+You have two options for authenticating the `gcloud` command:
+
+
+- You can create a **service account** within your GCP project. You can then
+[download a `.json` key file](https://cloud.google.com/iam/docs/creating-managing-service-account-keys) 
+associated with the account, and run the 
+[`gcloud auth activate-service-account`](https://cloud.google.com/sdk/gcloud/reference/auth/activate-service-account)
+command to authenticate your `gcloud` session.
+This is the recommended way to authenticate with GCP.
```

Comment:
I think this is a little confusing. Using a service account is recommended when accessing GCP from GCP; e.g. from a Kubeflow cluster.

Its not necessarily recommended when running on your local machine.